src/docs/product/accounts/membership.mdx

```diff
@@ -9,87 +9,116 @@ description: "Learn about the different types of Sentry membership, roles, and a
+| ------------------------------------------------------------------------------------------------------------------------ | -------------------- | -------------------------------------------------------------------- |
+| Can view and act on issues (such as assigning and resolving)                                                             | ✓                    | ✓                                                                    |
+| Can join and leave teams (depending on open membership settings)                                                         | ✓                    | ✓                                                                    |
+| Can add and remove repositories                                                                                          |                      | ✓ (_Only for teams they’re an existing admin for_)                   |
+| Can change project settings                                                                                              |                      | ✓ (_Only for projects they have access to_)                          |
+| Can assign Team Admin Role                                                                                               |                      | ✓ (_Only for teams they’re an existing admin for_)                   |
+| Can add, edit, or remove Team Contributors                                                                               |                      | ✓ (_Only for teams they’re an existing admin for_)                   |
+| Can add projects for teams                                                                                               | ✓                    | ✓ (_Only for teams they’re an existing admin for_)                   |
+| Can remove projects from teams                                                                                           |                      | ✓ (_Only for teams they’re an existing admin for_)                   |
+| Can invite new members to join the organization (_Invite will land in pending state for Managers and Owners to approve_) | ✓                    | ✓                                                                    |
+| Can remove an existing team from the organization                                                                        |                      | ✓ (_Only for teams they’re an existing admin for_)                   |
+| Can transer existing projects                                                                                            |                      | ✓ (_Only for projects owned by teams they’re an existing admin for_) |
+| Can remove existing projects from the organization                                                                       |                      | ✓ (_Only for projects owned by teams they’re an existing admin for_) |
```

Comment:
Want to double check that team admins can do this for their teams or if it's only org managers & owners than can

Reply:
Team admins will be able to delete projects. This is correct.